platforms/core-configuration/model-core/src/main/java/org/gradle/api/internal/provider/BuildableBackedProvider.java

```diff
@@ -45,12 +45,9 @@ public Class<T> getType() {
 
     @Override
     public ValueProducer getProducer() {
+        // not a lambda for readability purposes.
+        //noinspection Convert2Lambda
```

Comment:
TIL about IDEA's `//noinspection` comments, thanks. 

Not sure I agree this is more readable than the lambda version, but fine with me.